### ISSUE TITLE
Additional parent classes are forced to be on a new line always. Is this intentional?

### ISSUE THREAD

**Issue by ZakTaccardi** [pinterest/ktlint#1838]
Is reformatting:

```kts
import kotlinx.coroutines.CoroutineScope

internal abstract class MyParentClass(
    param1: String,
    param2: String
)

internal class MyChildClass(
    scope: CoroutineScope
) : MyParentClass(
    param1 = "one",
    param2 = "two"
), CoroutineScope by scope

```

to be:

```kts
import kotlinx.coroutines.CoroutineScope

internal abstract class MyParentClass(
    param1: String,
    param2: String
)

internal class MyChildClass(
    scope: CoroutineScope
) : MyParentClass(
    param1 = "one",
    param2 = "two"
),
    CoroutineScope by scope

```
a bug? the error is `Missing newline after ","` and violates the `wrapping` rule.

My take here is that the following format looks...off (bc it can fit on the same line):

```
),
    CoroutineScope by scope
```

Using version `0.48.2`

**Comment by paul-dingemans** [comment]
I am not sure whether to classify it as a bug or not. On one hand, I do agree that it looks less pretty than the original code. The argument `bc it can fit on the same line` is not a convincing argument as that depends on the size of your indent. Also the [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html#class-headers) do not provide an example that fits with your example. Based on the examples, I think that it should be formatted something like:
```
internal class MyChildClass(
    scope: CoroutineScope
) : MyParentClass(
        param1 = "one",
        param2 = "two"
    ),
    CoroutineScope by scope
```

This can best be resolved when implementing #1349.

**Comment by ZakTaccardi** [comment]
```kt
internal class MyChildClass(
    scope: CoroutineScope
) : MyParentClass(
        param1 = "one",
        param2 = "two"
    ),
    CoroutineScope by scope
```

actually, yeah, this looks good 👍

**Comment by paul-dingemans** [comment]
Implemented by #1349

**Comment by paul-dingemans** [comment]
See #2119
